File: fib.js
const {
    performance
} = require('perf_hooks');


function fib(n) {
    if (n <= 0) {
        return 1;
    }
    return fib(n - 1) + fib(n - 2);
}

let eo = -performance.now();
fib(35);
eo += performance.now();
//console.log(`TIME: ${eo}ms`);
console.log(`TIME: ${eo * 0.001}s`)

File: fib.go
package main

import (
	"fmt"
	"time"
)

func fib(n int) int {
	if n <= 1 {
		return n
	}
	return fib(n-1) + fib(n-2)
}

func main() {
	start := time.Now()
	fib(35)
	fmt.Println("TIME: ", time.Since(start).Seconds(), "s")
}